### ISSUE TITLE
Seeker of Knowledge (2800 Quests) Impossible (Horde Only)

### ISSUE THREAD

**Issue by GrizB37** [slowtorta/turtlewow-bug-tracker#6139]
As title, I guess this could be treated as a bug, as there's no way a Horde character can achieve that amount of completed quests. 

I made a forum post with feedback from horde players: https://forum.turtle-wow.org/viewtopic.php?f=4&t=12860

A few Alliance characters on the realm have the title, but every Horde completionist I've spoken to is stuck around 2250-2300, with all zones, seasonals and repeatables done. 

Basically I think the issue is Alliance have alot more quests to do, so it's only fair to reduce the horde requirement for 'Seeker of Knowledge' to around 2300.

Even if we had to wait for new patches / content to come we'd be waiting years. 

Thanks for your time.

**Comment by twowTurtReynolds** [comment]
This does not belong in the bug reports.
